daml/Marketplace/Distribution/Role.daml

```diff
@@ -20,6 +20,12 @@ template Role
         do
           create Auction.Offer with ..
 
+      nonconsuming ApproveAuctionServiceRequest : ContractId Auction.Service
```

Comment:
Same Q as in my previous comment. Didn't we have choices to approve auction requests previuosly? If not, how was it done? Or have you removed some other choice? I can't see one though.